### PR TITLE
docs: add contributing guide.

### LINKED ISSUE #22: [Docs] Contributing guide doc needed.
We need a comprehensive contributing guide for developers both in English and in Chinese.

The contributing guide can be like a traditional one but with the custom workflow of this project.

### PR DESCRIPTION
**Pull request by heming6666**
This PR adds contributing guide to website.

Please @frank-zsy  help review this if I miss something that need to be added. 

close #22 .

Signed-off-by: LinHaiming <lhming23@outlook.com>

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,153 @@
...
+
+-   Update your master with the latest upstream version:
+
+    ```shell
+    $ git pull --ff upstream master
+    ```
+
+## <a name="cla"></a> Signing the CLA
+
+Please sign our [Contributor License Agreement (CLA)][cla] before sending pull requests. For any code
```
Comment (frank-zsy) [inline docs/CONTRIBUTING.md:142]: We can remove the CLA part because we do not have CLA now and I don't think we need CLA for this project.

Comment (heming6666) [reply docs/CONTRIBUTING.md:142]: ok

**Comment by heming6666** [comment]
@frank-zsy I hava updated this PR.

**Pull request merged**